app/models/bus_stops_route.rb

```diff
@@ -0,0 +1,9 @@
+class BusStopsRoute < ApplicationRecord
+  belongs_to :bus_stop
+  belongs_to :route
+
+  validates :sequence, presence: true, numericality: { greater_than_or_equal_to: 1 }
+  validates :direction, presence: true
+  validates_uniqueness_of :sequence, scope: [ :route, :direction ]
```

Comment:
Shouldn't have spaces inside of square brackets.

Reply:
Whoops, got it.